Assets/_Project/Code/Scripts/QuestionStage/AnswerController.cs

```diff
@@ -0,0 +1,235 @@
+    }
+
+    public void CheckAnswer(string playerAnswer)
+    {
+        SetItemsInteractivity(false);
+        _isAnswerChecked = true;
+        CheckAnswerServerRpc(playerAnswer);
+    }
+
+    [ServerRpc(RequireOwnership = false)]
+    void CheckAnswerServerRpc(string playerAnswer)
+    {
+        if (currentQuestion.IsCorrect(playerAnswer))
+        {
+            SendFeedbackToClientsRpc("Brawo! Poprawna odpowiedź.");
+        }
+        else
+        {
+            SendFeedbackToClientsRpc($"Niestety, to nie jest poprawna odpowiedź. " +
+                $"Poprawne odpowiedzi to: {string.Join(", ", currentQuestion.CorrectAnswers)}");
+        }
+    }
+
+    [Rpc(SendTo.ClientsAndHost)]
+    void SendFeedbackToClientsRpc(string feedback)
+    {
+        feedbackText.text = feedback;
+        StartCoroutine(ChangeScene("Lobby", 4));
+    }
+
+    public void AskForHint()
+    {
+        UseHintNotifyServerRpc();
+    }
+
+    [ServerRpc(RequireOwnership = false)]
+    void UseHintNotifyServerRpc()
+    {
+        ShowHintRpc(currentQuestion);
+    }
+
+    [Rpc(SendTo.ClientsAndHost)]
+    void ShowHintRpc(Question question)
+    {
+        SetHintMode(true);
+        for (int i = 0; i < 4; i++)
+        {
+            answerButtons[i].GetComponentInChildren<TMP_Text>().text = question.Hints[i];
+        }
+    }
+    private void SetHintMode(bool active)
+    {
+        if (active)
+            SetButtonsDefaultColor();
+
+        answerInput.gameObject.SetActive(!active);
+
+        hintButtonsContainer.SetActive(active);
+    }
+
+    private void OnSelectButton(Button button)
+    {
+        SetButtonsDefaultColor();
+        button.GetComponent<Image>().color = Color.blue;
+        TMP_Text buttonText = button.GetComponentInChildren<TMP_Text>();
+        string buttonValue = buttonText.text;
+        CheckAnswer(buttonValue);
+    }
+
+    private void SetButtonsDefaultColor()
+    {
+        foreach (Button button in answerButtons)
+        {
+            button.GetComponent<Image>().color = Color.white;
+        }
+    }
+
+    private IEnumerator ChangeScene(string sceneName, float time)
+    {
+        yield return new WaitForSeconds(time);
+        NetworkManager.Singleton.SceneManager.LoadScene(sceneName, LoadSceneMode.Single);
+    }
+
+    private void SetItemsInteractivity(bool set)
+    {
+        foreach (Button button in answerButtons)
+        {
+            button.interactable = set;
+        }
+
+        answerInput.interactable = set;
+        useHintsButton.interactable = set;
+        submitButton.interactable = set;
+    }
+
+    [Rpc(SendTo.Everyone)]
+    public void SendQuestionToClientRpc(string questionText)
+    {
+        this.questionText.text = questionText;
+        _isAnswerChecked = false;
+        roundNumber.text = "Runda numer: " + currentQuestionIndex.ToString();
+        feedbackText.text = "";
+        answerInput.text = "";
+        timerText.text = "Czas: 30s";
```

Comment:
niepotrzebna linia, realizuje ją ShowCurrentTimeRpc, a jeśli zmieni się domyślny czas odpowiedzi to będzie tylko mylące